### DISCUSSION TITLE
HOW TO SET A MACRO TO REPEAT ON SOME SPECIFIC DAYS OF THE MONTH

### DISCUSSION THREAD

**Discussion by polin79** [WarmUpTill/SceneSwitcher#1185] [Q&A] [answered]
Hi,
Is it possible to set in the conditions to run a Macro on specific days of the month for example day 1, 2 or 3 of each month of the year. Because I have found only the possibility to set the days of the week, but not specific dates of the month.

Thanks
Paolo

**Comment by WarmUpTill** [comment, marked as answer]
This should be possible by using the "Date" condition type.
Click the "Show advanced settings" button and then select the "Pattern" option.

![](https://github.com/user-attachments/assets/9c722dbb-fd30-4ccf-b1c3-14b64cab5671)

Here you can specify the pattern (regular expression) to which the current date will be compared to.
The example above would be for the case "the first day of any month, of any year, at any time".

Hope that helps!

**Reply by polin79** [reply]
Thank very much: wonderful
